resources/web/wwi/nodes/utils/WbBoundingSphere.js

```diff
@@ -172,7 +173,7 @@ export default class WbBoundingSphere {
     if (!this.parentCoordinatesDirty)
       return;
 
-    if (this.transformOwner) {
+    if (this.#owner.nodeType === WbNodeType.WB_NODE_TRANSFORM) {
```

Comment:
Seems wrong to check for transform in BO

Reply:
I am not sure as bounding spheres are not the same as bounding objects. They are used for some visibility computation (at least in Webots), so they should probably consider the case where they are inside a Transform node. However, I am not sure if webots.js also use bounding spheres for this kind of purposes.

Reply:
As far as I know, the implementation of the bounding sphere in webots.js matches exactly the one in Webots.
So it seems correct to handle the transform case, as the bounding sphere is computed for geometries that can have Transform parents.

Reply:
Yes my bad, I went to fast and read bounding object instead bounding sphere. It seems correct.